components/landlord-setup/landlord-setup-flow.tsx

```
@@ -0,0 +1,156 @@
+interface PropertyData {
+	streetAddress?: string;
+	unitNumber?: string;
+	city?: string;
+	state?: string;
+	zip?: string;
+	propertyType?: string;
+	bedrooms?: string;
+	bathrooms?: string;
+	squareFootage?: string;
+	yearBuilt?: string;
+	[key: string]: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)